src/scheduler/fifo.c

```diff
@@ -1996,8 +2007,13 @@ find_ready_resv_job(resource_resv **resvs)
 	for (i = 0; resvs[i] != NULL && rjob == NULL; i++) {
 		if (resvs[i]->resv != NULL) {
 			if (resvs[i]->resv->resv_state ==RESV_RUNNING) {
-				if (resvs[i]->resv->resv_queue !=NULL)
-					rjob = find_runnable_resresv(resvs[i]->resv->resv_queue->jobs);
+				if (resvs[i]->resv->resv_queue !=NULL) {
+					ind = find_runnable_resresv(resvs[i]->resv->resv_queue->jobs, 0);
```

Comment:
I added _ind to the function names (find_non_normal_job() as well).  There is precedent to appending _ind instead of _index for these types of functions.